.github/workflows/sanitizer-tests.yaml

```
@@ -115,7 +115,7 @@ jobs:
     runs-on: ubuntu-22.04
 
     env:
-      FASTDDS_BRANCH: ${{ github.head_ref || github.event.inputs.fastdds_branch || 'master' }}
+      FASTDDS_BRANCH: ${{ github.event.inputs.fastdds_branch || 'master' }}
```

Should it not be `3.0.x-devel` branch if there is no input?

```suggestion
      FASTDDS_BRANCH: ${{ github.event.inputs.fastdds_branch || '3.0.x-devel' }}
```